src/metabase/api/dashboard.clj

```diff
@@ -683,40 +635,32 @@
       (throw (ex-info (tru "This dashboard has tab, makes sure every card has a tab")
                       {:status-code 400})))
     (api/check-500
-      (let [tabs-stats-atom (atom nil)]
+      (let [update-stats (atom nil)]
```

Comment:
I think this name could be improved because it starts with `update-` which implies it is a function, not a noun. How about `changes-stats` or `changes-info`? `changes` has the benefit of not being specific to update operations.
```suggestion
      (let [changes-stats (atom nil)]
```

Reply:
`changes` SGTM

Reply:
updated in c7a6e58f97